source/installation-guide/upgrading/different_major.rst

```diff
@@ -140,6 +140,29 @@ Upgrade Wazuh manager
 
       # apt-get install wazuh-api
 
+5. Disable the Wazuh repository:
```

Comment:
There's no step to "Enable the Wazuh repository". Any person who follows this guide will have the Wazuh repositories locked (from #493) but currently, there's no step to re-enable it.

Example: [Documentation](https://documentation.wazuh.com/current/installation-guide/upgrading/latest_wazuh3_minor.html#upgrade-to-the-latest-elastic-stack-version)